ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -645,7 +645,7 @@ protocolInfoCardano protocolParamsByron@ProtocolParamsByron {
     -- The major protocol version of the last era is the maximum major protocol
     -- version we support.
     maxMajorProtVer :: MaxMajorProtVer
-    maxMajorProtVer = MaxMajorProtVer (pvMajor protVerBabbage)
+    maxMajorProtVer = MaxMajorProtVer (pvMajor protVerConway)
```

Comment:
This will collide with PR #4111 when you rebase onto master.

However, the point of PR 4111 was to make it more obvious how to maintain this definition, so hopefully it'll be trivial to resolve the conflict.

Reply:
It was :)